### PR TITLE
Integrate refactors to dev: 25 March

### PR DESCRIPTION
**Pull request by marc-lopez**
@noelsison2 For review. Reopen #17 after merge

### CODE REVIEW

File: RRS/customer/urls.py [hunk 1]
```diff
@@ -10,7 +10,7 @@
         name  = 'index'
     ),
     url(
-        regex = r'restaurant/(\d+)$',
+        regex = r'restaurant/(\d+)/$',
```
Comment (noelsison) [inline RRS/customer/urls.py:13]: I'm checking the webpages open in my tabs now, but they only seem to use "/" for directories and without "/" for most urls

Comment (noelsison) [inline RRS/customer/urls.py:13]: Please ignore my earlier comment. Adding a trailing slash is the correct way to do it. The only time we don't add a slash is when the URL points to a file.

http://www.morethanseven.net/2009/02/10/append-slashes-urls-django/
http://webdesign.about.com/od/beginningtutorials/f/why-urls-end-in-slash.htm

**Comment by noelsison** [comment]
I'm assuming that the **helpers** app will contain all kinds of boilerplate code and will not be limited to tests. If this is correct, we should put boilerplate test code in `RRS/helpers/test.py`.

BTW, when I saw `RRS/helpers/tests.py`, I thought that our tests would have tests :))

**Comment by marc-lopez** [comment]
I will delete the unnecessary files. Right now, I can't think of other boilerplate code other than for tests, so I think the they can stay in _ _init_ _.py at the moment.

**Comment by noelsison** [comment]
Ok, will merge

**Pull request merged**